enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/PresenceCheckService.java

```diff
@@ -224,6 +214,38 @@ public void cleanup(OwnerId ownerId) throws PresenceCheckException, RemoteCommun
         }
     }
 
+    /**
+     * Init presence check and upload upscaled image.
+     *
+     * @param ownerId Owner identification.
+     * @param idVerification Verification identity.
+     * @throws DocumentVerificationException When not able to find documet image.
+     * @throws PresenceCheckException In case of business logic error.
+     * @throws RemoteCommunicationException In case of remote communication error.
+     */
+    private void initPresentCheckWithImage(final OwnerId ownerId, final IdentityVerificationEntity idVerification)
+            throws PresenceCheckException, DocumentVerificationException, RemoteCommunicationException {
+
+        if (!idVerification.isPresenceCheckInitialized()) {
+            if (StringUtils.hasText(idVerification.getSessionInfo())) {
```

Comment:
We should check the content of the `sessionInfo`, not only this presence, because this record is shared among all parts of the identity verification process. Currently we only use it for iProov, but that could change in the future.